src/olympia/migrations/1035-add-akismet-waffle.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO waffle_switch (name, active, note, created, modified)
+VALUES ('akismet', 1, 'Submit user content to Akismet', NOW(), NOW());
```

Comment:
Personal opinion, the switch name should explicitly mention what it does. In this example, I'd prefer `enable-akismet-spam-checking` as the flag name which will make reading code behaving differently much easier.

Reply:
I thought that was what `note` was for, heh.

Reply:
`note` is only visible in the admin though :see_no_evil:

Reply:
https://github.com/mozilla/addons-server/pull/9127/commits/8598eae9b3c47f11f11d46fc5a0ab8422e76e211